crash after updating Rocket and launching

After updating Rocke 3.3.0(7159).nightly, it would crash when I tried to launch it every time. 

Video: https://drive.google.com/open?id=1ywX9dSFXOPXZlEcvR9iAYp8m_WMCuTIA

Reproduce Rate 10/10

Pixel 2
Android 8.1.0
3.3.0(7195).nightly
Webview 69.0.3497.100

--------- beginning of crash
09-23 13:07:43.879 19615 19701 E AndroidRuntime: FATAL EXCEPTION: AsyncQueryWorker
09-23 13:07:43.879 19615 19701 E AndroidRuntime: Process: gro.allizom.zelda.beta, PID: 19615
09-23 13:07:43.879 19615 19701 E AndroidRuntime: java.lang.IllegalStateException: attempt to re-open an already-closed object: SQLiteDatabase: /data/user/0/gro.allizom.zelda.beta/databases/history.db
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.database.sqlite.SQLiteClosable.acquireReference(SQLiteClosable.java:55)
09--------- -23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1404)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1383)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.db.framework.FrameworkSQLiteDatabase.query(FrameworkSQLiteDatabase.java:161)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.db.framework.FrameworkSQLiteDatabase.query(FrameworkSQLiteDatabase.java:150)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.room.RoomOpenHelper.hasRoomMasterTable(RoomOpenHelper.java:151)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.room.RoomOpenHelper.checkIdentity(RoomOpenHelper.java:123)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.room.RoomOpenHelper.onOpen(RoomOpenHelper.java:115)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onOpen(FrameworkSQLiteOpenHelper.java:151)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:409)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:298)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:96)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.arch.persistence.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:54)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at org.mozilla.focus.provider.HistoryProvider.insert(HistoryProvider.java:73)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.content.ContentProvider$Transport.insert(ContentProvider.java:265)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.content.ContentResolver.insert(ContentResolver.java:1587)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.content.AsyncQueryHandler$WorkerHandler.handleMessage(AsyncQueryHandler.java:96)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
09-23 13:07:43.879 19615 19701 E AndroidRuntime: 	at android.os.HandlerThread.run(HandlerThread.java:65)
09-23 13:07:43.897  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:43.900   611   643 W SurfaceFlinger: Attempting to set client state on removed layer: Splash Screen gro.allizom.zelda.beta#0
09-23 13:07:43.900   611   643 W SurfaceFlinger: Attempting to destroy on removed layer: Splash Screen gro.allizom.zelda.beta#0
09-23 13:07:43.922  2589 19288 E NetworkScheduler: Invalid component specified.
09-23 13:07:43.942  2589 19288 E NetworkScheduler: Invalid component specified.
09-23 13:07:43.964  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:44.042 11015 11015 I StreamRenderer: cleanUpRendererScope(nowcards--4962016075009154925)
09-23 13:07:44.051 11015 19238 I PBSessionCacheImpl: Deleted sessionId[1527205280798] from persistence.
09-23 13:07:44.057 11015 11015 W SearchServiceCore: Abort, client detached.
09-23 13:07:44.059 11015 11015 I StreamController: cleanUpControllerScope(nowcards--4962016075009154925)
09-23 13:07:44.684  2589 19288 E NetworkScheduler: Invalid component specified.
09-23 13:07:44.807   841   875 I CHRE    : @ 241384.156: [AR_CHRE] still: 31
09-23 13:07:44.807   841   875 I CHRE    : @ 241384.156: [AR_CHRE] ON => IDLE
09-23 13:07:44.810  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:44.876   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 46000 mC, weight[0] 1
09-23 13:07:44.876   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 49000 mC, weight[1] -1
09-23 13:07:44.978  2589 19288 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.003  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.032  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.073  1165  2785 W ActivityManager:   Force finishing activity gro.allizom.zelda.beta/org.mozilla.focus.activity.MainActivity
09-23 13:07:45.077  1165  1205 I ActivityManager: Showing crash dialog for package gro.allizom.zelda.beta u0
09-23 13:07:45.119  1165  5481 I OpenGLRenderer: Initialized EGL, version 1.4
09-23 13:07:45.119  1165  5481 D OpenGLRenderer: Swap behavior 1
09-23 13:07:45.123 11015 11015 W SearchServiceClient: start new client, handoverId [1] client id [48446019757851518]
09-23 13:07:45.123 11015 11015 W SearchServiceClient: com.google.android.apps.gsa.shared.util.common.f: SearchServiceClient: start new client, handoverId [1] client id [48446019757851518]
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.shared.util.common.e.a(SourceFile:43)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.shared.util.common.e.f(SourceFile:6)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.service.SearchServiceClient.a(SourceFile:116)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.service.SearchServiceClient.startWithNewSession(SourceFile:103)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.overlay.a.b.aUJ(SourceFile:485)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.overlay.a.b.onResume(SourceFile:530)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.nowoverlayservice.b.onResume(SourceFile:295)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.g.PY(SourceFile:29)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.q.handleMessage(SourceFile:123)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.r.handleMessage(SourceFile:9)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at android.os.Handler.dispatchMessage(Handler.java:102)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at android.os.Looper.loop(Looper.java:193)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at android.app.ActivityThread.main(ActivityThread.java:6669)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at java.lang.reflect.Method.invoke(Native Method)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
09-23 13:07:45.123 11015 11015 W SearchServiceClient: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
09-23 13:07:45.123 11015 11015 W SessionLifecycleManager: Handover failed. Creating new session controller.
09-23 13:07:45.125   611   611 D SurfaceFlinger: duplicate layer name: changing com.google.android.apps.nexuslauncher/com.google.android.apps.nexuslauncher.NexusLauncherActivity to com.google.android.apps.nexuslauncher/com.google.android.apps.nexuslauncher.NexusLauncherActivity#1
09-23 13:07:45.128  2589 17595 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.145 11015 11015 I StreamController: registerScope: nowcards--4962016075009154925
09-23 13:07:45.157  1622  1622 I WallpaperService: engine resumed
09-23 13:07:45.165   841   875 D CHRE    : @ 241384.531: +: id 10, otherClientPresent 1, mode 3
09-23 13:07:45.166   841   875 D CHRE    : @ 241384.531: [ImuCal] Dynamic sensor configuration: high-performance.
09-23 13:07:45.166   841   875 D CHRE    : @ 241384.531: sensorType 9 allowed 1: mergedMode 3, otherClientPresent 1
09-23 13:07:45.166   841   875 D CHRE    : @ 241384.531: sensorType 12 allowed 1: mergedMode 3, otherClientPresent 1
09-23 13:07:45.168   841   875 D CHRE    : @ 241384.531: sensorType 14 allowed 0: mergedMode 3, otherClientPresent 0
09-23 13:07:45.182  1165  1204 I ActivityManager: Killing 18464:com.ss.android.ugc.trill/u0a154 (adj 906): empty #17
09-23 13:07:45.182 11015 11015 W TemplatedStringEvaluato: Got templated string with no display string
09-23 13:07:45.183 11015 11015 I ClusterController: Found empty cluster, or no children successfully created.
09-23 13:07:45.183  1165  1207 W libprocessgroup: kill(-18464, 9) failed: No such process
09-23 13:07:45.196  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.200 11015 11015 I StreamRenderer: maybeRegisterNowCardsScope(nowcards--4962016075009154925, com.google.android.sidekick.shared.remoteapi.CardRenderingContext@732cfd0)
09-23 13:07:45.225  1165  1207 W libprocessgroup: kill(-18464, 9) failed: No such process
09-23 13:07:45.238  1165  2737 D ConnectivityService: ConnectivityService NetworkRequestInfo binderDied(NetworkRequest [ LISTEN id=255, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED&FOREGROUND Unwanted:  Uid: 10154] ], android.os.BinderProxy@2b7d05f)
09-23 13:07:45.238 11015 11015 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
09-23 13:07:45.239 11015 11015 I chatty  : uid=10047 com.google.android.googlequicksearchbox:search identical 1 line
09-23 13:07:45.240 11015 11015 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
09-23 13:07:45.240   712   712 I Zygote  : Process 18464 exited due to signal (9)
09-23 13:07:45.240 11015 11015 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
09-23 13:07:45.256  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.269  1165  1207 W libprocessgroup: kill(-18464, 9) failed: No such process
09-23 13:07:45.269  1165  1207 I libprocessgroup: Successfully killed process cgroup uid 10154 pid 18464 in 85ms
09-23 13:07:45.273   739  1128 D sound_trigger_hw: stdev_start_recognition:[1] Enter
09-23 13:07:45.273   739  1128 D sound_trigger_hw: get_confidence_levels_payload: No smlib for uuid, sending opaque data
09-23 13:07:45.273   739  1128 D sound_trigger_hw: loaded_state_fn:[1] handle event id 2
09-23 13:07:45.273   739  1128 D sound_trigger_platform: platform_stdev_get_capture_device: available devices 0x80000004, device 0x80000004
09-23 13:07:45.273   739  1128 D sound_trigger_platform: platform_stdev_get_device_sample_rate: sample rate 48000
09-23 13:07:45.273   739  1128 D sound_trigger_platform: platform_stdev_send_hwmad_cal: previous acdb_id 128 matches new acdb_id 128 return
09-23 13:07:45.273   739  1128 D sound_trigger_platform: platform_stdev_send_calibration: sending device calibration for device(1) acdb_id(128)
09-23 13:07:45.273   739  1128 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 128, Topology Id 12e19
09-23 13:07:45.274   739  1128 D sound_trigger_hw: set_device: enable device (1) = listen-cpe-handset-mic
09-23 13:07:45.274   739  1128 D audio_route: Apply path: listen-cpe-handset-mic
09-23 13:07:45.280  1165  2806 I ActivityManager: Process com.android.chrome:sandboxed_process0 (pid 19670) has died: cch  CEM 
09-23 13:07:45.280  1165  2806 W ActivityManager: Scheduling restart of crashed service gro.allizom.zelda.beta/org.chromium.content.app.SandboxedProcessService0 in 1000ms
09-23 13:07:45.280  1165  1207 W libprocessgroup: Failed to open process cgroup uid 99040 pid 19670: No such file or directory
09-23 13:07:45.280  1165  1207 E libprocessgroup: Error encountered killing process cgroup uid 99040 pid 19670: No such file or directory
09-23 13:07:45.280 19615 19666 W cr_ChildProcessConn: onServiceDisconnected (crash or killed by oom): pid=19670
09-23 13:07:45.286  1525  1525 I Zygote  : Process 19670 exited cleanly (0)
09-23 13:07:45.293  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:45.295   739  1128 D sound_trigger_hw: start:[1] gcs start VA engine, enable 1 token 38
09-23 13:07:45.295   739  1128 D sound_trigger_hw: start:[1] calling gcs_enable with handle 1, non-persist cal 0xe91424e8, sz 24
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CALIBRATION_SIZE
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CALIBRATION
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CALIBRATION_SIZE
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CALIBRATION
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CONNECTION_SIZE
09-23 13:07:45.295   739  1128 E         : gcs_get_sg_mod_conn_and_update_pld: subgraph mod conn size 0. Exiting
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CONNECTION_SIZE
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_MODULE_CONNECTION
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_CONNECTIONS_SIZE
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPHS_CONNECTION
09-23 13:07:45.295   739  1128 D ACDB-LOADER: ACDB -> ACDB_WDSP_CMD_GET_SUBGRAPH_CONNECTIONS_SIZE
09-23 13:07:45.295   739  1128 E         : __gcs_get_sg_conn_and_update_pld: subgraph conn size 0. Exiting
09-23 13:07:45.306   739  1128 D sound_trigger_hw: reg_hal_event_session: ST_EVENT_SESSION_REGISTER capture_handle 62 p_ses 0xe8f8a880
09-23 13:07:45.306   739  1128 D sound_trigger_hw: session[1]: loaded_state_fn ---> active_state_fn 
09-23 13:07:45.306   739  1128 D sound_trigger_hw: stdev_start_recognition:[1] Exit
09-23 13:07:45.306  1165  1678 I SoundTriggerHelper: startRecognition successful.
09-23 13:07:45.307  2430  2430 I GsaVoiceInteractionSrv: Start Hotword Recognition Status: true
09-23 13:07:45.320 11015 11025 I earchbox:searc: Background concurrent copying GC freed 160013(6MB) AllocSpace objects, 1(108KB) LOS objects, 48% free, 25MB/49MB, paused 156us total 159.438ms
09-23 13:07:45.579 11015 19605 I EventLogSendingHelper: Sending log events.
09-23 13:07:45.840  1165  2818 D WificondControl: Scan result ready event
09-23 13:07:45.859  1165  1337 D WificondScannerImpl: Filtering out 4 scan results.
09-23 13:07:45.878   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 47000 mC, weight[0] 1
09-23 13:07:45.878   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 47000 mC, weight[1] -1
09-23 13:07:46.368 19615 19656 I CrashlyticsCore: Crashlytics report upload complete: 5BA71F9F01C4-0001-4C9F-52DA1DE65958
09-23 13:07:46.879   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 47000 mC, weight[0] 1
09-23 13:07:46.880   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 46000 mC, weight[1] -1
09-23 13:07:46.907 19615 19656 I CrashlyticsCore: Crashlytics report upload complete: 5BA71F8E02BA-0001-4B5F-52DA1DE65958
09-23 13:07:47.886   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 46000 mC, weight[0] 1
09-23 13:07:47.886   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 45000 mC, weight[1] -1
09-23 13:07:48.219  1165  2785 I ActivityManager: START u0 {act=android.intent.action.APP_ERROR flg=0x10000000 cmp=com.android.vending/com.google.android.finsky.activities.AppCrashProxy (has extras)} from uid 1000
09-23 13:07:48.219  1165  5481 D OpenGLRenderer: endAllActiveAnimators on 0x6f7fbd9300 (RippleDrawable) with handle 0x6f94036f20
09-23 13:07:48.238   820   820 D android.hardware.power@1.2-service.wahoo-libperfmgr: LAUNCH ON
09-23 13:07:48.244 19615 19701 I Process : Sending signal. PID: 19615 SIG: 9
09-23 13:07:48.246 29848 29848 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@9f09eee
09-23 13:07:48.267 29848 29848 I AppLifecycleTracker: onActivityCreated: 
09-23 13:07:48.268 29848 29848 I Finsky  : [2] com.google.android.finsky.activities.AppCrashProxy.onCreate(44): Sending feedback for crashed gro.allizom.zelda.beta to com.google.android.gms
09-23 13:07:48.269  1165  2785 I ActivityManager: START u0 {act=android.intent.action.APP_ERROR flg=0x10000000 pkg=com.google.android.gms cmp=com.google.android.gms/.feedback.FeedbackActivity (has extras)} from uid 10014
09-23 13:07:48.273   820   820 D android.hardware.power@1.2-service.wahoo-libperfmgr: LAUNCH ON
09-23 13:07:48.276  1165  3586 D ConnectivityService: ConnectivityService NetworkRequestInfo binderDied(NetworkRequest [ LISTEN id=266, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED&FOREGROUND Unwanted:  Uid: 10143] ], android.os.BinderProxy@ff71ff8)
09-23 13:07:48.277  1165  1207 W libprocessgroup: kill(-19615, 9) failed: No such process
09-23 13:07:48.277  1165  1207 I libprocessgroup: Successfully killed process cgroup uid 10143 pid 19615 in 0ms
09-23 13:07:48.277  1165  1837 I ActivityManager: Process gro.allizom.zelda.beta (pid 19615) has died: cch  CRE 
09-23 13:07:48.278   707   707 I Zygote  : Process 19615 exited due to signal (9)
09-23 13:07:48.281  1165  1215 W ActivityManager: setHasOverlayUi called on unknown pid: 19615
09-23 13:07:48.286 19563 19563 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@baf3aee
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: Failed to instantiate custom view inflater android.support.v7.app.AppCompatViewInflater. Falling back to default.
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: java.lang.ClassNotFoundException: android.support.v7.app.AppCompatViewInflater
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.Class.classForName(Native Method)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.Class.forName(Class.java:453)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.Class.forName(Class.java:378)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.support.v7.app.AppCompatDelegateImpl.a(:com.google.android.gms@13280037@13.2.80 (100400-211705629):1)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.support.v7.app.AppCompatDelegateImpl.onCreateView(:com.google.android.gms@13280037@13.2.80 (100400-211705629):1)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:772)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.policy.DecorView.onResourcesLoaded(DecorView.java:1885)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.policy.PhoneWindow.generateLayout(PhoneWindow.java:2599)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.policy.PhoneWindow.installDecor(PhoneWindow.java:2672)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.policy.PhoneWindow.getDecorView(PhoneWindow.java:2071)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.support.v7.app.AppCompatDelegateImpl.n(:com.google.android.gms@13280037@13.2.80 (100400-211705629):8)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.support.v7.app.AppCompatDelegateImpl.p(:com.google.android.gms@13280037@13.2.80 (100400-211705629):1)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.support.v7.app.AppCompatDelegateImpl.c(:com.google.android.gms@13280037@13.2.80 (100400-211705629):1)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at xrg.a(:com.google.android.gms@13280037@13.2.80 (100400-211705629):115)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.google.android.gms.feedback.FeedbackChimeraActivity.onCreate(:com.google.android.gms@13280037@13.2.80 (100400-211705629):8)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.google.android.chimera.Activity.publicOnCreate(Unknown Source:0)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at dij.onCreate(:com.google.android.gms@13280037@13.2.80 (100400-211705629):6)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at nzp.onCreate(:com.google.android.gms@13280037@13.2.80 (100400-211705629):2)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.Activity.performCreate(Activity.java:7136)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.Activity.performCreate(Activity.java:7127)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2893)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.os.Handler.dispatchMessage(Handler.java:106)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.os.Looper.loop(Looper.java:193)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at android.app.ActivityThread.main(ActivityThread.java:6669)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.reflect.Method.invoke(Native Method)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.app.AppCompatViewInflater" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/system/framework/com.android.media.remotedisplay.jar", zip file "/system/framework/com.android.location.provider.jar", zip file "/data/app/com.google.android.gms-gbYibe064zc-EHQuv4L48w==/base.apk"],nativeLibraryDirectories=[/data/app/com.google.android.gms-gbYibe064zc-EHQuv4L48w==/lib/arm64, /data/app/com.google.android.gms-gbYibe064zc-EHQuv4L48w==/base.apk!/lib/arm64-v8a, /system/lib64]]
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
09-23 13:07:48.295 19563 19563 I AppCompatDelegate: 	... 37 more
09-23 13:07:48.317  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:48.345   739  1128 D sound_trigger_hw: stdev_stop_recognition:[1] Enter
09-23 13:07:48.345   739  1128 D sound_trigger_hw: active_state_fn:[1] handle event id 7
09-23 13:07:48.345   739  1128 D sound_trigger_hw: active_state_fn:[1] unhandled event
09-23 13:07:48.345   739  1128 D sound_trigger_hw: active_state_fn:[1] handle event id 4
09-23 13:07:48.345   739  1128 D sound_trigger_hw: dereg_hal_event_session: ST_EVENT_SESSION_DEREGISTER capture_handle 62 p_ses 0xe8f8a880
09-23 13:07:48.345   739  1128 D sound_trigger_hw: stop:[1] gcs stop VA engine, size=24
09-23 13:07:48.353   739  1128 D sound_trigger_hw: stop:[1] calling gcs_disable with handle 1
09-23 13:07:48.353   739  1128 E         : __gcs_disable_update_pld: sgt_array[i] = 0x1
09-23 13:07:48.353   739  1128 E         : __gcs_disable_update_pld: sgt_array[i] = 0x2
09-23 13:07:48.361   739  1128 D sound_trigger_hw: set_device: disable device (1) = listen-cpe-handset-mic
09-23 13:07:48.364   739  1128 D sound_trigger_hw: session[1]: active_state_fn ---> loaded_state_fn 
09-23 13:07:48.364   739  1128 D sound_trigger_hw: stdev_stop_recognition:[1] Exit status 0
09-23 13:07:48.367  2430  2430 I GsaVoiceInteractionSrv: Stop Hotword Recognition status: true
09-23 13:07:48.372  1165  1215 I ActivityManager: Displayed com.google.android.gms/.feedback.FeedbackActivity: +84ms (total +125ms)
09-23 13:07:48.377   820   820 D android.hardware.power@1.2-service.wahoo-libperfmgr: LAUNCH OFF
09-23 13:07:48.382   611   643 W SurfaceFlinger: Attempting to set client state on removed layer: Dim Layer for - mAboveAppWindowsContainers#0
09-23 13:07:48.382   611   643 W SurfaceFlinger: Attempting to destroy on removed layer: Dim Layer for - mAboveAppWindowsContainers#0
09-23 13:07:48.383   611  2107 W SurfaceFlinger: Attempting to destroy on removed layer: ab37a6d Application Error: gro.allizom.zelda.beta#0
09-23 13:07:48.387  1469  1469 W VoiceImeTranscriptor: UIME feature enabled: true
09-23 13:07:48.389  1469  1469 I Conv2QueryExtension: Current locale: en_US, config allows these locales: de,en,fr,it,es,pt
09-23 13:07:48.389  1469  1469 I Conv2QueryExtension: Conv2Query not enabled due to current app [com.google.android.gms] not in whitelist
09-23 13:07:48.389  1469  1469 I Conv2QueryExtension: onActivate() : Disabled by unsupported host app
09-23 13:07:48.389  1469  1469 W VoiceImeTranscriptor: UIME feature enabled: true
09-23 13:07:48.407  1469  1469 I FeaturePermissions: onSharedPreferenceChanged() : Key = pref_key_latest_ime_activation_time : Feature = null
09-23 13:07:48.408  1469  1469 I FeaturePermissions: onSharedPreferenceChanged() : Key = latest_activation_time : Feature = null
09-23 13:07:48.408  1469  1469 I PhenotypeExpConfig: refreshConfiguration() : Force = false : UpdateAvailable = false : Age = 70 minutes : MaxAge = 720 minutes
09-23 13:07:48.431  1469  1469 I LatinIme: onActivate() : EditorInfo = Package = com.google.android.gms : Type = Text : Learning = Enable : Suggestion = Show : AutoCorrection = Enable : Microphone = Show : NoPersonalizedLearning = Disable
09-23 13:07:48.444  1469  5716 E native  : personalized-decoder.cc:769 Cannot create graph rescoring model.
09-23 13:07:48.446  1469  1469 W VoiceImeTranscriptor: UIME feature enabled: true
09-23 13:07:48.446  1469  1469 W VoiceImeTranscriptor: UIME feature enabled: true
09-23 13:07:48.447  1469  1469 I Conv2QueryExtension: Current locale: en_US, config allows these locales: de,en,fr,it,es,pt
09-23 13:07:48.448  1469  1469 I Conv2QueryExtension: Conv2Query not enabled due to current app [com.google.android.gms] not in whitelist
09-23 13:07:48.448  1469  1469 I Conv2QueryExtension: onActivate() : Disabled by unsupported host app
09-23 13:07:48.448  1469  1469 W VoiceImeTranscriptor: UIME feature enabled: true
09-23 13:07:48.464  1469  1469 I FeaturePermissions: onSharedPreferenceChanged() : Key = pref_key_latest_ime_activation_time : Feature = null
09-23 13:07:48.465  1469  1469 I FeaturePermissions: onSharedPreferenceChanged() : Key = latest_activation_time : Feature = null
09-23 13:07:48.535   611   642 W SurfaceFlinger: Attempting to set client state on removed layer: Splash Screen com.google.android.gms#0
09-23 13:07:48.535   611   642 W SurfaceFlinger: Attempting to destroy on removed layer: Splash Screen com.google.android.gms#0
09-23 13:07:48.599  1622  1622 I WallpaperService: engine paused
09-23 13:07:48.603   841   875 D CHRE    : @ 241387.969: -: id 10, otherClientPresent 0, mode 3
09-23 13:07:48.604   841   875 D CHRE    : @ 241387.969: [ImuCal] Dynamic sensor configuration: stand-by.
09-23 13:07:48.604   841   875 D CHRE    : @ 241387.969: sensorType 9 allowed 1: mergedMode 3, otherClientPresent 1
09-23 13:07:48.605   841   875 D CHRE    : @ 241387.969: sensorType 12 allowed 1: mergedMode 3, otherClientPresent 1
09-23 13:07:48.607 29848 29848 I AppLifecycleTracker: onActivityDestroyed: 
09-23 13:07:48.608   841   875 D CHRE    : @ 241387.969: sensorType 14 allowed 0: mergedMode 3, otherClientPresent 0
09-23 13:07:48.610   611   643 W SurfaceFlinger: Attempting to destroy on removed layer: AppWindowToken{7d794e0 token=Token{f02e3 ActivityRecord{a152b12 u0 com.android.vending/com.google.android.finsky.activities.AppCrashProxy t2514}}}#0
09-23 13:07:48.756  2589 19289 E NetworkScheduler: Invalid component specified.
09-23 13:07:48.778 11015 11015 I StreamRenderer: cleanUpRendererScope(nowcards--4962016075009154925)
09-23 13:07:48.785 11015 19238 I PBSessionCacheImpl: Deleted sessionId[1527205280799] from persistence.
09-23 13:07:48.791 11015 11015 W SearchServiceCore: Abort, client detached.
09-23 13:07:48.793 11015 11015 I StreamController: cleanUpControllerScope(nowcards--4962016075009154925)
09-23 13:07:48.887   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 46000 mC, weight[0] 1
09-23 13:07:48.887   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 46000 mC, weight[1] -1
09-23 13:07:49.603   841   875 I CHRE    : @ 241388.953: [AR_CHRE] IDLE => ON
09-23 13:07:49.890   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 45000 mC, weight[0] 1
09-23 13:07:49.890   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 45000 mC, weight[1] -1
09-23 13:07:50.045  2640  2640 D SetupWizard: [DeferredPriorityHelper] Deferred notification was dismissed previously.
09-23 13:07:50.045  2640  2640 I SetupWizard: [DeferredSetupNotificationSchedulerService] Removing existing notification. Priority is DO_NOT_SHOW
09-23 13:07:50.604 11015 19278 I EventLogSendingHelper: Sending log events.
09-23 13:07:50.657 11015 19604 I WorkController: WorkProxy is not enqueued because WorkController is disposed: WorkProxy{Name=context::h, WorkerId=context, id=6e2d3ed}
09-23 13:07:50.657 11015 19604 I WorkController: WorkProxy is not enqueued because WorkController is disposed: WorkProxy{Name=context::j, WorkerId=context, id=174a222}
09-23 13:07:50.659 11015 19604 I WorkController: WorkProxy is not enqueued because WorkController is disposed: WorkProxy{Name=context::k, WorkerId=context, id=5b94b3}
09-23 13:07:50.659 11015 19604 I WorkController: WorkProxy is not enqueued because WorkController is disposed: WorkProxy{Name=context::i, WorkerId=context, id=3bf0a70}
09-23 13:07:50.659 11015 19604 I WorkController: WorkProxy is not enqueued because WorkController is disposed: WorkProxy{Name=context::m, WorkerId=context, id=25c1e9}
09-23 13:07:50.715  2589  3975 I Places  : ?: PlacesBleScanner start() with priority 2
09-23 13:07:50.721  2589  3975 I Places  : ?: PlacesBleScanner start() with priority 2
09-23 13:07:50.721  2589  3975 I PlaceInferenceEngine: [anon] Changed inference mode: 1
09-23 13:07:50.721  2589  3975 I PlaceInferenceEngine: [account#-1795736791] Changed inference mode: 1
09-23 13:07:50.725  2589  3975 I Places  : Converted 0 out of 18 WiFi scans
09-23 13:07:50.731  2589  3975 I Places  : ?: PlacesBleScanner start() with priority 2
09-23 13:07:50.733  2589  3975 I PlaceInferenceEngine: [anon] Changed inference mode: 1
09-23 13:07:50.734  2589  3975 I Places  : Converted 0 out of 18 WiFi scans
09-23 13:07:50.752  2589 17595 I PlaceInferenceEngine: No beacon scan available - ignoring candidates.
09-23 13:07:50.823  2589 17595 I PlaceInferenceEngine: No beacon scan available - ignoring candidates.
09-23 13:07:50.891   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 45000 mC, weight[0] 1
09-23 13:07:50.891   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 46000 mC, weight[1] -1
09-23 13:07:51.894   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 44000 mC, weight[0] 1
09-23 13:07:51.894   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 44000 mC, weight[1] -1
09-23 13:07:52.896   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 44000 mC, weight[0] 1
09-23 13:07:52.896   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 43000 mC, weight[1] -1
09-23 13:07:53.100  1165  2785 I ActivityManager: Killing 18415:com.google.android.apps.maps/u0a84 (adj 906): empty #17
09-23 13:07:53.117  1165  1207 W libprocessgroup: kill(-18415, 9) failed: No such process
09-23 13:07:53.120  1165  6117 I ActivityManager: Killing 18032:com.google.process.gapps/u0a21 (adj 906): empty #17
09-23 13:07:53.140  1165  6117 D ConnectivityService: ConnectivityService NetworkRequestInfo binderDied(NetworkRequest [ LISTEN id=256, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Unwanted:  Uid: 10084] ], android.os.BinderProxy@7a35683)
09-23 13:07:53.142   707   707 I Zygote  : Process 18032 exited due to signal (9)
09-23 13:07:53.142   707   707 I Zygote  : Process 18415 exited due to signal (9)
09-23 13:07:53.159  1165  1207 W libprocessgroup: kill(-18415, 9) failed: No such process
09-23 13:07:53.159  1165  1207 I libprocessgroup: Successfully killed process cgroup uid 10084 pid 18415 in 42ms
09-23 13:07:53.160  1165  1207 W libprocessgroup: kill(-18032, 9) failed: No such process
09-23 13:07:53.160  1165  1207 I libprocessgroup: Successfully killed process cgroup uid 10021 pid 18032 in 0ms
09-23 13:07:53.897   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 43000 mC, weight[0] 1
09-23 13:07:53.897   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 43000 mC, weight[1] -1
09-23 13:07:54.898   734   821 I ThermalEngine: vs_get_temperature: read[0] tsens_tz_sensor13 43000 mC, weight[0] 1
09-23 13:07:54.898   734   821 I ThermalEngine: vs_get_temperature: read[1] tsens_tz_sensor0 42000 mC, weight[1] -1
09-23 13:07:54.963  1165  1206 I ActivityManager: Start proc 19774:com.google.android.gms.feedback/u0a21 for service com.google.android.gms/.feedback.FeedbackAsyncService
09-23 13:07:54.963 19774 19774 E id.gms.feedbac: Not starting debugger since process cannot load the jdwp agent.
09-23 13:07:54.984 19774 19774 I id.gms.feedbac: The ClassLoaderContext is a special shared library.
09-23 13:07:54.986 19774 19774 I chatty  : uid=10021 com.google.android.gms.feedback identical 1 line
09-23 13:07:54.989 19774 19774 I id.gms.feedbac: The ClassLoaderContext is a special shared library.
09-23 13:07:55.007 11015 11015 W SearchServiceClient: start new client, handoverId [1] client id [48446019757851518]
09-23 13:07:55.007 11015 11015 W SearchServiceClient: com.google.android.apps.gsa.shared.util.common.f: SearchServiceClient: start new client, handoverId [1] client id [48446019757851518]
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.shared.util.common.e.a(SourceFile:43)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.shared.util.common.e.f(SourceFile:6)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.service.SearchServiceClient.a(SourceFile:116)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.service.SearchServiceClient.startWithNewSession(SourceFile:103)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.overlay.a.b.aUJ(SourceFile:485)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.search.shared.overlay.a.b.onResume(SourceFile:530)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.apps.gsa.nowoverlayservice.b.onResume(SourceFile:295)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.g.PY(SourceFile:29)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.q.handleMessage(SourceFile:123)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.google.android.libraries.gsa.d.b.r.handleMessage(SourceFile:9)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at android.os.Handler.dispatchMessage(Handler.java:102)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at android.os.Looper.loop(Looper.java:193)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at android.app.ActivityThread.main(ActivityThread.java:6669)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at java.lang.reflect.Method.invoke(Native Method)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
09-23 13:07:55.007 11015 11015 W SearchServiceClient: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
09-23 13:07:55.010   611   611 D SurfaceFlinger: duplicate layer name: changing com.google.android.apps.nexuslauncher/com.google.android.apps.nexuslauncher.NexusLauncherActivity to com.google.android.apps.nexuslauncher/com.google.android.apps.nexuslauncher.NexusLauncherActivity#1
09-23 13:07:55.014 11015 11015 I SearchServiceCore: Event Bus thread is disabled; marking main thread as EventBus
09-23 13:07:55.020  1469  1469 I PhenotypeExpConfig: refreshConfiguration() : Force = false : UpdateAvailable = false : Age = 70 minutes : MaxAge = 720 minutes
09-23 13:07:55.022  1469  1469 I LatinIme: onDeactivate()
09-23 13:07:55.024 11015 11015 W SessionLifecycleManager: Handover failed. Creating new session controller.
09-23 13:07:55.026  1622  1622 I WallpaperService: engine resumed
09-23 13:07:55.031 19774 19774 V NativeCrypto: Registering com/google/android/gms/org/conscrypt/NativeCrypto's 284 native methods...
09-23 13:07:55.032   841   875 D CHRE    : @ 241394.406: +: id 10, otherClientPresent 1, mode 3
09-23 13:07:55.033   841   875 D CHRE    : @ 241394.406: [ImuCal] Dynamic sensor configuration: high-performance.
09-23 13:07:55.033   841   875 D CHRE    : @ 241394.406: sensorType 9 allowed 1: mergedMode 1, otherClientPresent 1
09-23 13:07:55.034   841   875 D CHRE    : @ 241394.406: sensorType 12 allowed 1: mergedMode 1, otherClientPresent 1
09-23 13:07:55.035   841   875 D CHRE    : @ 241394.406: sensorType 14 allowed 0: mergedMode 3, otherClientPresent 0
09-23 13:07:55.047 11015 11015 I StreamController: registerScope: nowcards--4962016075009154925
09-23 13:07:55.050 19774 19774 D NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: false
09-23 13:07:55.055 19774 19774 I ProviderInstaller: Installed default security provider GmsCore_OpenSSL
09-23 13:07:55.070 11015 11015 W TemplatedStringEvaluato: Got templated string with no display string
09-23 13:07:55.071 11015 11015 I ClusterController: Found empty cluster, or no children successfully created.
09-23 13:07:55.087 11015 19236 W LocationOracle: No location history returned by ContextManager
09-23 13:07:55.095 11015 11015 I StreamRenderer: maybeRegisterNowCardsScope(nowcards--4962016075009154925, com.google.android.sidekick.shared.remoteapi.CardRenderingContext@732cfd0)
09-23 13:07:55.102  1165  5481 I OpenGLRenderer: Initialized EGL, version 1.4
09-23 13:07:55.102  1165  5481 D OpenGLRenderer: Swap behavior 1
09-23 13:07:55.109 19774 19774 E id.gms.feedbac: The String#value field is not present on Android versions >= 6.0
09-23 13:07:55.120 19774 19782 W System  : A resource failed to call close. 
09-23 13:07:55.123 19774 19774 W Primes  : Primes not initialized, returning default (no-op) Primes instance which will ignore all calls. Please call Primes.initialize(...) before using any Primes API.

--- a/app/schemas/org.mozilla.rocket.persistance.History.HistoryDatabase/3.json
+++ b/app/schemas/org.mozilla.rocket.persistance.History.HistoryDatabase/3.json
@@ -1,0 +1,72 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 3,
+    "identityHash": "ed1350002e11ab61b6b9f2aab52e8ce4",
+    "entities": [
+      {
+        "tableName": "browsing_history",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`_id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, `title` TEXT, `url` TEXT NOT NULL, `view_count` INTEGER NOT NULL, `last_view_timestamp` INTEGER NOT NULL, `fav_icon_uri` TEXT)",
+        "fields": [
+          {
+            "fieldPath": "id",
+            "columnName": "_id",
+            "affinity": "INTEGER",
+            "notNull": true
+          },
+          {
+            "fieldPath": "title",
+            "columnName": "title",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "url",
+            "columnName": "url",
+            "affinity": "TEXT",
+            "notNull": true
+          },
+          {
+            "fieldPath": "viewCount",
+            "columnName": "view_count",
+            "affinity": "INTEGER",
+            "notNull": true
+          },
+          {
+            "fieldPath": "lastViewTimestamp",
+            "columnName": "last_view_timestamp",
+            "affinity": "INTEGER",
+            "notNull": true
+          },
+          {
+            "fieldPath": "favIconUri",
+            "columnName": "fav_icon_uri",
+            "affinity": "TEXT",
+            "notNull": false
+          }
+        ],
+        "primaryKey": {
+          "columnNames": [
+            "_id"
+          ],
+          "autoGenerate": true
+        },
+        "indices": [
+          {
+            "name": "index_browsing_history_view_count",
+            "unique": false,
+            "columnNames": [
+              "view_count"
+            ],
+            "createSql": "CREATE  INDEX `index_browsing_history_view_count` ON `${TABLE_NAME}` (`view_count`)"
+          }
+        ],
+        "foreignKeys": []
+      }
+    ],
+    "setupQueries": [
+      "CREATE TABLE IF NOT EXISTS room_master_table (id INTEGER PRIMARY KEY,identity_hash TEXT)",
+      "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, \"ed1350002e11ab61b6b9f2aab52e8ce4\")"
+    ]
+  }
+}

--- a/app/src/main/java/org/mozilla/rocket/persistance/History/HistoryDatabase.java
+++ b/app/src/main/java/org/mozilla/rocket/persistance/History/HistoryDatabase.java
@@ -25,7 +25,7 @@ import java.util.Map;
 // Current plan is to build something like the Place system in Firefox 3
 // https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Places/Database
 
-@Database(entities = {Site.class}, version = 2)
+@Database(entities = {Site.class}, version = 3)
 public abstract class HistoryDatabase extends RoomDatabase {
 
     private static final String CREATE_TABLE_IF_NOT_EXISTS = "CREATE TABLE IF NOT EXISTS ";
@@ -44,7 +44,7 @@ public abstract class HistoryDatabase extends RoomDatabase {
                 if (instance == null) {
                     instance = Room.databaseBuilder(context.getApplicationContext(),
                             HistoryDatabase.class, "history.db")
-                            .addMigrations(MIGRATION_1_2)
+                            .addMigrations(MIGRATION_1_2, MIGRATION_2_3)
                             .build();
                 }
             }
@@ -107,6 +107,17 @@ public abstract class HistoryDatabase extends RoomDatabase {
             } finally {
                 database.endTransaction();
             }
+        }
+    };
+
+    private static final Migration MIGRATION_2_3 = new Migration(2, 3) {
+
+        private static final String CREATE_INDEX = "CREATE INDEX IF NOT EXISTS index_browsing_history_view_count ON " +
+                HistoryContract.TABLE_NAME + "(" + HistoryContract.BrowsingHistory.VIEW_COUNT + ")";
+
+        @Override
+        public void migrate(SupportSQLiteDatabase database) {
+            database.execSQL(CREATE_INDEX);
         }
     };
 }

